.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,98 @@
+on:
+  workflow_call:
+    inputs:
+      artefact:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+
+jobs:
+  deploy:
+    name: Build, push, & deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      # region Docker setup
+      - name: Download image artefact
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artefact }}
+
+      # Load the image to make use of common layers during the final build.
+      - name: Load image from archive
+        run: docker load -i ${{ inputs.artefact }}.tar
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN  }}
+      # endregion
```

Comment:
oh.. I get it now. It's an IDE accordion. I don't know if that's neccessary in a short and readable workflow such as this one?

Reply:
Removed in e08183e